eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -142,7 +142,7 @@ jobs:
     CMOCKA_XML_FILE: "%g-test-results.xml"
     CMOCKA_MESSAGE_OUTPUT: "xml"
     AZURE_ENABLE_STATIC_ANALYSIS: 1
-    AZURE_KEYVAULT_AVOID_THROTTLED: 1
+    AZURE_KEYVAULT_AVOID_THROTTLED: 10
```

Comment:
Don't pay attention to this change.  
This is a workaround to be removed soon.

Currently, during live mode, each Key Vault test will wait `10` seconds between each test.  This change uses the actual value `10` as the time to delay the test. ( before this, the value `1` would hardcode 10 seconds all the time).

However, I am working to remove this approach and instead, only the test cases that really require to wait would introduce the delay ( like all the crypto-params tests ).  This will improve the total run time from test cases.

KeyVault has been updated.  Only Secrets and Certs are pending to be reviewed to remove this approach